jenkins/data-prepper/release-data-prepper-all-artifacts.jenkinsfile

```diff
@@ -25,6 +25,19 @@ pipeline {
                 }
             }
         }
+        stage('Download Maven Artifacts') {
+            steps {
+                script {
+                    mavenPath = "${DATA_PREPPER_ARTIFACT_STAGING_SITE}/${VERSION}/${DATA_PREPPER_BUILD_NUMBER}/maven"
+                    group = "org/opensearch/dataprepper"
+                    artifacts = ["data-prepper-api"]
+                    fileTypes = ["-javadoc.jar", ".jar", ".pom", "-sources.jar", ".module"]
+                    checksums = ["", ".md5", ".sha1", ".sha256", ".sha512"]
```

Comment:
why the empty string?

Reply:
Example in previous comment explains this.

Reply:
The empty string is the artifact file - it is not a checksum file.